sympy/matrices/matrices.py

```diff
@@ -3966,6 +3966,72 @@ def vech(self, diagonal=True, check_symmetry=True):
                     count += 1
         return v
 
+    def upper(a, diag=0):
+        """
```

Comment:
You should add 1 line summaries for each docs.

Reply:
Okay I will